### PR TITLE
映射文档 No. 322/323/333

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/PaConvert/issues/112

322 torch.Tensor.corrcoef
323 torch.nn.GLU
333 torch.special.polygamma

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## [仅 paddle 参数更多]torch.Tensor.corrcoef
+
+### [torch.Tensor.corrcoef](https://pytorch.org/docs/1.13/generated/torch.Tensor.corrcoef.html#torch.Tensor.corrcoef)
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md]: 这个不要用1.13版本torch的API，要用最新版本torch的API

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## [仅参数名不一致]torch.nn.GLU
+
+### [torch.nn.GLU](https://pytorch.org/docs/1.13/generated/torch.nn.GLU.html#torch.nn.GLU)
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 这个用最新版本torch的API是不是好一点

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.polygamma.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+## [ torch 参数更多 ]torch.special.polygamma
+
+### [torch.special.polygamma](https://pytorch.org/docs/1.13/special.html#torch.special.polygamma)
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.polygamma.md]: 这个不要用1.13版本torch的API，要用最新版本torch的API

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+```python
+torch.nn.GLU(dim=-1)
+```
+
+### [paddle.nn.functional.glu](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/glu_cn.html)
+
+```python
+paddle.nn.functional.glu(x, axis=-1, name=None)
+```
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 这个有没有Layer对应的API，使用functional是不是有点不合适。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+两者功能一致且参数用法一致，仅参数名不同，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                   |
+| ------- | ------------ | -------------------------------------- |
+| dim     | axis         | 沿着该轴将输入二等分，仅参数名不一致。 |
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 补一下转写示例

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 看到其他文档参数名不一致是没有转写示例

https://github.com/PaddlePaddle/docs/blob/4a97080f2dc079c0501dcf54b4b2b7f6365f325a/docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveMaxPool1d.md 
图片

https://github.com/PaddlePaddle/docs/blob/4a97080f2dc079c0501dcf54b4b2b7f6365f325a/docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.adaptive_avg_pool3d.md
图片

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md (hunk 4 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 图片

搜索没有其他对应API，那就是功能缺失了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md (hunk 1 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md (hunk 2 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.polygamma.md (hunk 3 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.polygamma.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md (hunk 5 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 这个写一下好一点，不写也没问题

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md (hunk 2 above)
Comment (ROckDog22) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 这个代码转换之前对应的torch版本是1.13，现在对应2.0版本的torch，所以用新的API好一点

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 可以再在目录下全部替换

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md (hunk 5 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 那先不写和其他文档一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+### [torch.Tensor.corrcoef](https://pytorch.org/docs/stable/generated/torch.Tensor.corrcoef.html#torch.Tensor.corrcoef)
+
+```python
+torch.Tensor.corrcoef()
+```
+
+### [paddle.linalg.corrcoef](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/linalg/corrcoef_cn.html#paddle.linalg.corrcoef)
+
+```python
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md:11]: 已有 tests/test_corrcoef.py torch.corrcoef 转为 paddle.linalg.corrcoef

torch.Tensor.corrcoef 转为 paddle.Tensor.corrcoef

这里是不是有点不一样，这里torch.Tensor.corrcoef和paddle.linalg.corrcoef对应

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md:11]: 这里是因为 paddle.Tensor.corrcoef 没有文档和对应代码行

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md:11]: 代码中是paddle.linalg.corrcoef，Tensor对象引用其他模块

**Review by ROckDog22 — APPROVED** [review]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+## [仅 paddle 参数更多]torch.Tensor.corrcoef
+
+### [torch.Tensor.corrcoef](https://pytorch.org/docs/stable/generated/torch.Tensor.corrcoef.html#torch.Tensor.corrcoef)
+
+```python
+torch.Tensor.corrcoef()
+```
+
+### [paddle.linalg.corrcoef](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/linalg/corrcoef_cn.html#paddle.linalg.corrcoef)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.corrcoef.md]: 这个对应到 `paddle.Tensor.corrcoef` ，文档没有是另一个问题，可以先写个()的链接。不能因为文档bug改变映射关系

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+## [仅参数名不一致]torch.nn.GLU
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 这个按功能缺失来吧，paddle应该在paddle.nn.functional.glu基础上再封装一层

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.GLU.md]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM
